### DISCUSSION TITLE
Screen lag when using keyboard

### DISCUSSION THREAD

**Discussion by Spoicyz** [ppy/osu#29180] [Q&A]
### Type

Game behaviour

### Bug description

When i press the keyboard button the game freezes for a second and then resumes all the actions that happened during the freeze a few seconds after

### Screenshots or videos

https://github.com/user-attachments/assets/03fc6bde-7859-44fd-b7ff-59e938d87355

### Version

2024.727.0-lazer

### Logs

[compressed-logs.zip](https://github.com/user-attachments/files/16401001/compressed-logs.zip)

**Comment by bdach** [comment]
Please upload video of this with frame graph shown (ctrl-f11 twice).
